app/Http/Controllers/Api/TalksController.php

```diff
@@ -0,0 +1,22 @@
+<?php namespace Symposium\Http\Controllers\Api;
+
+use Auth;
+use Talk;
+
+class TalksController extends BaseController
+{
+    /**
+     * Display the specified resource.
+     *
+     * @param  string $id
+     * @return Response
+     */
+    public function show($id)
+    {
+        $talk = Talk::where('id', $id)
+                ->where('author_id', Auth::user()->id)
+                ->firstOrFail();
```

Comment:
Whoops. I remember us noting that elsewhere. Once again this is code I just copied and pasted from elsewhere in the codebase. Great note. :)